### PR TITLE
🐋 feat: docker base image

### PR DESCRIPTION
**Pull request by florian-hoenicke**
We install commonly used libraries on the base image instead of building them every time again and again.
This saves 30 seconds per debugging iteration. So, if it makes 10 iterations, we saved 5 min.
When we add more dependencies to our base image, this will be even more impactful.

### CODE REVIEW

File: src/options/generate/static_files/base_image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM jinaai/jina:3.14.1-py39-standard
```
Comment (joschkabraun) [inline src/options/generate/static_files/base_image/Dockerfile:1]: we are using a different jina version now

Comment (joschkabraun) [reply src/options/generate/static_files/base_image/Dockerfile:1]: `FROM jinaai/jina:3.15.1-dev14-py39-standard`

**Review by joschkabraun — APPROVED** [review]
Only jina version of image needs an update, otherwise LGTM

File: src/options/generate/static_files/base_image/Dockerfile (hunk 1 above)
Comment (florian-hoenicke) [reply src/options/generate/static_files/base_image/Dockerfile]: done thanks

**Pull request merged**